### PR TITLE
Update instructions for setting up the toolchain

### PR DESCRIPTION
**Pull request by guoyr**
cc @niharika01

### CODE REVIEW

File: src/lamplib/tasks/download.py [hunk 1]
```diff
@@ -58,13 +58,11 @@ def fetch_and_install(self):
                         "internal volume your Mac. Verify this assumption with `diskutil list`, substituting the "
                         "appropriate volume name if required. \n\n"
 
-                        "echo 'data' | sudo tee -a /etc/synthetic.conf\n"
+                        sudo mkdir /opt/data
+                        sudo chown $USER /opt/data
+                        echo -e 'data\topt/data' | sudo tee -a /etc/synthetic.conf
```
Comment (guoyr) [inline src/lamplib/tasks/download.py:63]: SO doesn't do a great job at showing the link for an answer 😬  I had intended to point to the one with 44 votes, not the one after it. lol

https://apple.stackexchange.com/questions/371908/how-to-make-root-volume-writeable-again-in-catalina/372500#372500

Comment (richardsamuels) [inline src/lamplib/tasks/download.py:63]: This block should read:
```
"sudo mkdir /opt/data\n"
"sudo chown $USER /opt/data\n"
"echo -e 'data\topt/data' | sudo tee -a /etc/synthetic.conf\n\n"```

File: src/lamplib/tasks/download.py [hunk 2]
```diff
@@ -58,13 +58,11 @@ def fetch_and_install(self):
                         "internal volume your Mac. Verify this assumption with `diskutil list`, substituting the "
                         "appropriate volume name if required. \n\n"
```
Comment (richardsamuels) [inline src/lamplib/tasks/download.py:59]: With the removal of the calls to diskutil, lines 57-59 are no longer relevant. Move the \n\n from line 59 to line 56

**Comment by rtimmons** [comment]
This has languished, and users have not complained about the current setup instructions. In the interest of clearing out old genny PRs, I'm going to close without prejudice - if this still wants to be done please feel free to re-open and rebase/merge on to master.

**Pull request closed without merging**